app/kubernetes/helpers/application/index.js

```diff
@@ -28,6 +28,43 @@ class KubernetesApplicationHelper {
     return _.filter(pods, { Labels: app.spec.selector.matchLabels });
   }
 
+  static associateContainerPersistedFoldersAndConfigurations(app, containers) {
+    _.forEach(containers, (container) => {
+      container.PersistedFolders = _.without(
+        _.map(app.PersistedFolders, (pf) => {
+          if (pf.MountPath && _.includes(_.map(container.VolumeMounts, 'mountPath'), pf.MountPath)) {
```

Comment:
Extract `_.map(container.VolumeMounts, 'mountPath')` as var as it's used more than 1 time